### ISSUE TITLE
Logging to another PSHost produces unnecessary line breaks (Powershell Universal)

### ISSUE THREAD

**Issue by Callidus2000** [PowershellFrameworkCollective/psframework#540]
Hi,
I'm using PSFramework for many parts of my scripts and modules, especially the logging framework. Currently I'm facing the problem within (the fantastic) Powershell Universal (3.2.5) and how the script output is parsed and presented within the admin console, created [an issue for Adam from this](https://github.com/ironmansoftware/issues/issues/1516). He responded that they have a custom PSHost that captures this output and that he will take a look at it.

Meanwhile I'd be already happy to configure the output of the logging framework, e.g. to only include the message and skip the prefixed info. Is this possible to configure?

-Sascha

**Comment by FriedrichWeinmann** [comment]
Sorry about the delayed response, was on a lengthy vacation.
This is due to the color system requiring multiple Write-Host statements with `-NoNewLine`

You can remove the extra information with these settings:

```powershell
Set-PSFConfig PSFramework.Message.Style.FunctionName $false
Set-PSFConfig PSFramework.Message.Style.Timestamp $false
```

The next release will also include an option to strip out the coloring, in which case it will be but a single line, extra data or no extradata

**Comment by FriedrichWeinmann** [comment]
Released/Resolved in 1.7.244
Set `PSFramework.Message.Style.NoColor` to true and you should get a single line per message
